SMT-LIB/parse_all.py

```diff
@@ -53,27 +54,39 @@ def main():
     parser.add_argument('--out', type=str, default="stats.out", nargs='?',
                         help='Where to save the statistics')
 
+    parser.add_argument('--reps', type=int, default=1,
+                        help="number of times each benchmark is run")
+
     args = parser.parse_args()
 
     random.seed(42)
-    p = multiprocessing.Pool()
     chunks = multiprocessing.cpu_count()
     file_list = list(get_all_smt_files(args.base))
     random.shuffle(file_list)
     if args.count == -1:
         files_cnt = len(file_list)
     else:
         files_cnt = args.count
-    print("Submitting %d jobs, %d at the time" % (files_cnt, chunks))
-    timings = p.map(execute_script_fname, islice(file_list, files_cnt), chunks)
+
+    timings = []
+    errors = []
+
+    with multiprocessing.Pool() as p:
+        print(f"Submitting {files_cnt} jobs, {chunks} at the time")
+        for el in p.starmap(execute_script_fname,
+                            zip(islice(file_list, files_cnt),
+                                repeat(args.reps)),
+                            chunks):
+            (timings if el[0] > 0 else errors).append(el)
 
     mean = sum(x[0] for x in timings) / len(timings)
-    print("The mean execution time was %0.2f seconds" % mean)
-    print("The max execution time was %0.2f seconds" % max(x[0] for x in timings))
+    print(f"The mean execution time was {mean:.2f} seconds")
+    print(f"The max execution time was {max(x[0] for x in timings):.2f} seconds")
+    print(f"Found {len(errors)} errors: {[el[1] for el in errors]}")
```

Comment:
```suggestion
    print(f"Found {len(errors)} errors in : {[x[1] for x in errors]}")
```

We are using `el` above as the return of a single call. In the line above we use `x` as iterator. I'd use `x` here too for consistency.